ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-feature-operator-refactor.yaml

```diff
@@ -11,27 +7,30 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.18
+    tag: golang-1.21
```

Comment:
I think, we can have go version config from opendatahub-operator repo to sync latest version instead changing go version every time in CI job. 
I've already created a file `.ci-operator.yaml` earlier. 
```
build_root_image:
  namespace: openshift
  name: release
  tag: golang-1.21
```
Above config will pick by CI job from  `.ci-operator.yaml` file. https://github.com/opendatahub-io/opendatahub-operator/blob/incubation/.ci-operator.yaml

Image_stream_tage should replace by below code 

```
build_root:
  from_repository: true
```
@VaishnaviHire wdyt ?